File: Modul 6/SOAL 2/C/PRAK602.c
#include <stdio.h>

int main() {
    int nilai, elemen;
    scanf("%d", &nilai); //nilai sebagai jumlah ruangan

    int banyak_zetsu[nilai]; 
    
    for(int i = 0; i < nilai; i++) {
        scanf("%d", &elemen); 
        banyak_zetsu[i] = elemen;
    }

    printf("\n"); 

    for (int i = 0; i < nilai; i++) {
        int total = banyak_zetsu[i] * (1+i);
        printf("%d ", total);
    }
    return 0;
}

File: Modul 6/SOAL 2/PYTHON/PRAK602.py
nilai = int(input()) #nilai sebagai jumlah ruangan
elemen = list(map(int, input().split()))

banyak_zetsu = elemen

print()

for i in range (nilai):
    total = banyak_zetsu[i] * (1+i)
    print(total, end = " ")